ground/gcs/src/plugins/config/configstabilizationwidget.cpp

```diff
@@ -73,9 +73,50 @@ ConfigStabilizationWidget::ConfigStabilizationWidget(QWidget *parent) : ConfigTa
     disableMouseWheelEvents();
 
     connect(this,SIGNAL(autoPilotConnected()),this,SLOT(applyRateLimits()));
-}
 
 
+    // at first save the rate max values for the horizon rate plot
+
+    int roll_expo = m_stabilization->rateRollExpo->value();
+    int pitch_expo = m_stabilization->ratePitchExpo->value();
+    int yaw_expo = m_stabilization->rateYawExpo->value();
+    int roll_max = m_stabilization->fullStickRateRoll->value();
+    int pitch_max = m_stabilization->fullStickRatePitch->value();
+    int yaw_max = m_stabilization->fullStickRateYaw->value();
+    m_stabilization->rateStickExpoPlot->init(0,0,roll_expo,pitch_expo,yaw_expo,roll_max,pitch_max,yaw_max,0,0,0);
```

Comment:
Spacing after comma.  Also, is there any way you can call showExpoPlot below-- or factor its contents out-- to eliminate code duplication?